src/main/resources/logback.xml

```diff
@@ -6,6 +6,8 @@
 	<appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
 		<encoder class="net.logstash.logback.encoder.LogstashEncoder">
 			<includeMdcKeyName>X-Correlation-Id</includeMdcKeyName>
```

Comment:
we should change this to use `correlationId` label so it matches other services.